static/app/views/starfish/types.tsx

```diff
@@ -26,6 +26,17 @@ export enum SpanMetricsFields {
   SPAN_SELF_TIME = 'span.self_time',
 }
 
+export type SpanMetricsFieldTypes = {
```

Comment:
I think we want to prefer interfaces whenever possible due to https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections

Reply:
Is it completely whenever possible? I thought we were mostly using it for things like props that get extended. I'm happy to change in a future iteration, though!

Reply:
I've just been defaulting to `interface`, but I think we can do this whenever